src/nagini_translation/lib/context.py

```diff
@@ -45,6 +45,8 @@ def __init__(self) -> None:
         self._global_counter = 0
         self.perm_factor = None
         self._old_aliases = {}
+        self.current_thread_object = None
```

Comment:
I'm aware of the irony since none of the existing fields have any documentation whatsoever, but could you add a comment that explains what these new ones do or represent?